ocean_provider/utils/util.py

```diff
@@ -10,25 +10,22 @@
 from cgi import parse_header
 
 import requests
+from artifacts import ERC721Template
```

Comment:
I think all artifact imports should be after jsonsempai

Reply:
you're right.  isort got me.